lib/sensu-plugins-slack/version.rb

```diff
@@ -2,7 +2,7 @@ module SensuPluginsSlack
   module Version
     MAJOR = 1
     MINOR = 5
-    PATCH = 0
+    PATCH = 1
```

Comment:
please leave the versioning to maintainers as there is no guarantee on the order of released changes.